README.md

```diff
@@ -10,6 +10,41 @@ If this is your first time here, try [experimenting with the live demo](http://b
 Installation
 ------------
 
+We have provided a Vagrantfile for easy installation.  Vagrant is software that controls some of the major virtualization software such as [VMWare Workstation](https://www.vmware.com/products/workstation.html) and [Oracle VirtualBox](https://www.virtualbox.org/).  With a simple text file, you can clone a machine to run locally on your computer.  More details can be found at https://www.vagrantup.com. 
```

Comment:
These instructions should be moved under `With Vagrant`, as they are specific to that section. Also, the vagrant hyperlink at the end should be actually linked, i.e. `[https://www.vagrantup.com](https://www.vagrantup.com)`